### PR TITLE
Add Configure overloads for Auth for config/name

### PR DESCRIPTION
**Pull request by HaoK**
cc @Praburaj

### CODE REVIEW

File: src/Microsoft.AspNet.Authentication.Facebook/FacebookServiceCollectionExtensions.cs [hunk 1]
```diff
@@ -4,6 +4,7 @@
 using Microsoft.AspNet.Authentication.Facebook;
 using Microsoft.Framework.OptionsModel;
 using System;
+using Microsoft.Framework.ConfigurationModel;
```
Comment (Praburaj) [inline src/Microsoft.AspNet.Authentication.Facebook/FacebookServiceCollectionExtensions.cs]: sort namespaces.

**Comment by Praburaj** [comment]
Looks good to me :shipit:

**Comment by muratg** [comment]
:shipit:

**Comment by HaoK** [comment]
Merged 5e7f1d7eff060aea0010b32b8bf68b36b51635b2

**Pull request closed without merging**